docs/types/operators.rst

```diff
@@ -26,6 +26,18 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+.. note::
+    The result of the ternary operator (``... ? ... : ...``) is never a number literal, even if all of its operands are number literals.
+    The result type is the bigger of the mobile types of the numeric operands.
```

Comment:
```suggestion
    The result type is determined from the types of the two operands in the same way as above, converting to their mobile type first if required.
```

Reply:
The main reason to rephrase this is that "size" is difficult because of signed types.